sqlconnect/internal/databricks/integration_test.go

```diff
@@ -49,6 +49,31 @@ func TestDatabricksDB(t *testing.T) {
 		)
 	})
 
+	t.Run("with oauth", func(t *testing.T) {
+		oauthConfigJSON, ok := os.LookupEnv("DATABRICKS_OAUTH_TEST_ENVIRONMENT_CREDENTIALS")
+		if !ok {
+			if os.Getenv("FORCE_RUN_INTEGRATION_TESTS") == "true" {
+				t.Fatal("DATABRICKS_OAUTH_TEST_ENVIRONMENT_CREDENTIALS environment variable not set")
+			}
+			t.Skip("skipping databricks integration test due to lack of a test environment")
```

Comment:
Since we are not skipping entire test only the oAuth one.
```suggestion
			t.Skip("skipping databricks oAuth integration test due to lack of a test environment")
```